src/app/core/store/organizations/contact/contacts.effects.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {Injectable} from "@angular/core";
+import {catchError, map, mergeMap, tap} from "rxjs/operators";
+import {Actions, Effect, ofType} from "@ngrx/effects";
+import {Action, Store} from "@ngrx/store";
+import {Router} from "@angular/router";
+import {ContactConverter} from "./contact.converter";
+import {AppState} from "../../app.state";
+import {OrganizationService} from "../../../rest";
+import {NotificationsAction} from "../../notifications/notifications.action";
+import {Observable} from "rxjs/Observable";
+import {I18n} from "@ngx-translate/i18n-polyfill";
+import {ContactsAction, ContactsActionType} from "./contacts.action";
+
+@Injectable()
+export class ContactsEffects {
+  @Effect()
+  public getContact$: Observable<Action> = this.actions$.pipe(
+    ofType<ContactsAction.GetContact>(ContactsActionType.GET_CONTACT),
+    mergeMap(action => this.organizationService.getOrganizationContact(action.payload.organizationId)),
```

Comment:
Method assumes `code` not `id`.

Reply:
Sure, this is just a wrong name of the attribute in the object. It is a copy paste error, I must fix that!